installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc

```diff
@@ -0,0 +1,8 @@
+[id="ipi-install-configuration-files"]
+= Configuration Files
+include::modules/common-attributes.adoc[]
```

Comment:
The Configuration files section is not building correctly when I build it locally. You already have the include::modules/common-attributes.adoc[] part included in the assembly that you're placing this content in. So, I would try removing include::modules/common-attributes.adoc[] from this section to see if it renders correctly. Same thing for the 4.3 content in the separate PR. Right now, the build is displaying the raw contents of the common-attributes.adoc before showing the modules.